scrapers_next/or/committee.py

```diff
@@ -0,0 +1,61 @@
+from spatula import XPath, URL, HtmlListPage, HtmlPage, CSS
+from openstates.models import ScrapeCommittee
+import re
+
+class CommitteeDetail(HtmlListPage):
+    source = URL("https://olis.oregonlegislature.gov/liz/2023R1/Committees/JCT/CommitteeMembers")
+    selector = CSS('tbody tr')
+
+    def process_item(self, item):
+        com = self.input
+        position, name = item.getchildren()
+        name = re.sub(r"Senator |Representative ", "", name.text_content().strip())
+        position = position.text_content().strip()
+        com.add_member(name, position)
+
+        return com
+
+
+class CommitteeList(HtmlListPage):
+    source = "https://olis.oregonlegislature.gov/liz/Committees/list/"
+
+    def process_item(self, item):
+        link = item.get("href").split('/')
+        link = 'https://olis.oregonlegislature.gov/liz/' + link[4] +\
+                "/Committees/" + link[6] + "/CommitteeMembers"
+
+        try:
+            parent = self.parent
+        except:
+            parent = None
+
+        com = ScrapeCommittee(
+                name=item.text_content(),
+                chamber=self.chamber,
+                classification=self.classification,
```

Comment:
Rather than specifying classification as an attribute of the `CommitteeList` class, you can just rely on the fact that interdependency of `parent` and `classification` here. If there is not truthy value for `parent`, then it must be a committee and not a subcommittee.

Therefore you can rewrite this line as:
`classification="subcommittee" if parent else "committee",`

And then just get remove wherever you specified `classification` for classes at the bottom of the page.